outputs.tf

```diff
@@ -26,6 +26,16 @@ output "storage_account_network_rules" {
   )
 }
 
+output "storage_account_primary_blob_endpoint" {
```

Comment:
This is not needed for your added feature "customer_managed_key" and this endpoint is already available through the `storage_account_properties` output